plugins/modules/purefb_s3acc.py

```diff
@@ -227,6 +285,28 @@ def create_s3acc(module, blade):
                     msg="Failed to set quotas correctly for account {0}. "
                     "Error: {1}".format(module.params["name"], res.errors[0].message)
                 )
+        if PUBLIC_API_VERSION in versions:
+            if not module.params["block_new_public_policies"]:
+                module.params["block_new_public_policies"] = False
+            if not module.params["block_public_access"]:
+                module.params["block_public_access"] = False
+            osa = ObjectStoreAccountPatch(
+                public_access_config=PublicAccessConfig(
+                    block_new_public_policies=module.params[
+                        "block_new_public_policies"
+                    ],
+                    block_public_access=module.params["block_public_access"],
+                )
+            )
+            res = blade2.patch_object_store_accounts(
```

Comment:
`blade2` hasn't be initialized at this point. This causes account creation to fail with an error like:
```
  File "/var/folders/jw/mbnzt6gs5637b_npdxx7m15c0000gn/T/ansible_purestorage.flashblade.purefb_s3acc_payload_l4axqygq/ansible_purestorage.flashblade.purefb_s3acc_payload.zip/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3acc.py", line 403, in <module>
  File "/var/folders/jw/mbnzt6gs5637b_npdxx7m15c0000gn/T/ansible_purestorage.flashblade.purefb_s3acc_payload_l4axqygq/ansible_purestorage.flashblade.purefb_s3acc_payload.zip/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3acc.py", line 397, in main
  File "/var/folders/jw/mbnzt6gs5637b_npdxx7m15c0000gn/T/ansible_purestorage.flashblade.purefb_s3acc_payload_l4axqygq/ansible_purestorage.flashblade.purefb_s3acc_payload.zip/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3acc.py", line 301, in create_s3acc
UnboundLocalError: cannot access local variable 'blade2' where it is not associated with a value
```
Setting this beforehand fixes it:
```python
blade2 = get_system(module)
```

Reply:
Please try this new parth: #319

Reply:
Tested and works.

Reply:
@jeremy 1.19.1 has been released with all fixes.